report_thesis/src/sections/experiments/optimization_experiment/optimization_experiment_design.tex

```diff
@@ -0,0 +1,40 @@
+\subsection{Optimization Experiment Design}\label{subsec:optimization_experiment_design}
+Using the remaining ten models, we conducted an extended experiment to further refine their performance for each oxide. 
+The goal was to identify which preprocessing techniques and hyperparameters would yield the best performance for each model by doing a thorough search for each configuration. 
+To achieve this, we evaluated multiple permutations of each model with various preprocessors and hyperparameter configurations. 
+Each configuration included a mandatory scaler, while data transformation and dimensionality reduction techniques were optional. 
+The optimization process was conducted using our optimization framework, outlined in Section~\ref{sec:optimization_framework}
+
+To obtain a fair assessment of each configuration, we needed to strike a balance between doing enough iterations for the optimization to converge and what was feasible given the time constraints. 
+Therefore, we decided to do 200 iterations per model for each oxide, totaling 16,000 iterations given ten models and eight oxides. 
+We deemed this to be a reasonable number of iterations to get a good indication of the performance of each configuration.
+As mentioned in Section~\ref{sec:optimization_framework}, we used the \gls{tpe} algorithm for the optimization process.
+For this sampler, we set the number of startup trials to 25\%.
+The number of startup trials controls the number of random samples that are drawn before the \gls{tpe} sampler engages. 
+By choosing 25\%, we would reserve the first quarter of the iterations for exploration.
+We believed that this would allow the sampler enough time to explore the search space while still providing enough iterations for refinement. 
```

Comment:
```suggestion
We believed this approach would allow sufficient time for the sampler to explore the search space while still providing enough iterations for refinement.
```